src/admin-portal/create-campaign/campaign-testimonial-view.js

```diff
@@ -0,0 +1,234 @@
+                      opt === show?.tabOne && "test-show-opt"
+                    }`}
+                    onClick={() => {
+                      setShow({ ...opt, tabOne: opt });
+
+                      if (opt === "All") {
+                        setData(testimonials);
+                      }
+                    }}
+                  >
+                    {opt}
+                  </button>
+                );
+              })}
+            </div>
+          </Col>
+        </Row>
+
+        <div>
+          {allopts?.map((chx) => {
+            return (
+              <Row key={chx?.name} className="mt-5">
+                <Col>
+                  <div>
+                    <div className="flex items-center justify-content-between">
+                      <button
+                        className="text-black bg-transparent text-muted text-lg tracking-wide"
+                        onClick={() => {
+                          if (show?.tabTwo !== chx?.name) {
+                            setShow({ ...show, tabTwo: chx?.name });
+                          } else if (show?.tabTwo === chx?.name) {
+                            setShow({ ...show, tabTwo: null });
+                          }
+                        }}
+                      >
+                        {chx?.name}
+                        <span
+                          className={`bg-gradient px-3 tracking-wide ${
+                            show?.tabTwo === chx?.name && "rotate-180"
+                          }`}
+                        >
+                          <svg
+                            width="12"
+                            height="7"
+                            viewBox="0 0 12 7"
+                            fill="none"
+                            xmlns="http://www.w3.org/2000/svg"
+                          >
+                            <path
+                              d="M10.59 0.589996L6 5.17L1.41 0.589996L0 2L6 8L12 2L10.59 0.589996Z"
+                              fill="#2D2D2D"
+                            />
+                          </svg>
+                        </span>
+                      </button>
+                      {/* {chx?.platform === "campaign" && ( */}
+                      <button
+                        onClick={() => {
+                          setShow({ ...show, showFormFor: chx?.platform });
+                          setOpenModal(true);
+                        }}
+                        className=" px-3 bg-white py-2 flex items-center gap-3 cursor-pointer adtxt"
+                      >
+                        <span>
+                          <FontAwesomeIcon icon={faAdd} />
+                        </span>
+                        {chx?.addtext}
+                      </button>
+                      {/* )} */}
+                    </div>
+
+                    <div>
+                      {show?.tabTwo === chx?.name && (
```

Comment:
Please ensure you use descriptive names for variables. `chx` on line 171 for example tells nothing about what it represents or what is being stored there.
Avoid cryptic names. They make reading code hard and hard to maintain